### DISCUSSION TITLE
Question regarding language codes

### DISCUSSION THREAD

**Discussion by jaschaurbach** [bookwyrm-social/bookwyrm#2829] [General]
While I was going through settings.py I found this:

https://github.com/bookwyrm-social/bookwyrm/blob/290b74039297349693f4f139fa58659a19d1e1ac/bookwyrm/settings.py#L297

now I am more than surprised because Esperanto has only `es` as ISO-Code. `es-uy` would be "Esperanto spoken in Uruguay". The correct code should be `es-es` but maybe there is a reason for this. @mouse-reeve could you clarify? That'll would be great!

**Comment by bpeel** [comment]
`es-es` would be the Spanish spoken in Spain. Did you mean to say that the correct code should be `eo-eo`? I think that would be weird too, because there is no country for Esperanto (it’s meant as an international language) and `eo` probably isn’t a valid country code. If the country code is mandatory I’ve sometimes seen locales like `eo_XX`, but ideally you’d just leave the country code out and have a locale called `eo`.

**Reply by jaschaurbach** [reply]
Oh sorry, I meant eo. My bad. Yes, best thing would be to just use eo and nothing else. 

I would just change this but I do not know how this works out together with the localization from crowdin. the localization has eo_UY in the name https://github.com/bookwyrm-social/bookwyrm/tree/l10n_main/locale/eo_UY/LC_MESSAGES

**Reply by mouse-reeve** [reply]
I used eo_UY because Crowdin provided it, but now that I look around, it doesn't seem to be particularly standard. I think we could change it to just "eo" without too much trouble if it makes a difference?
